Js/modal_header.js

```diff
@@ -0,0 +1,28 @@
+const open_modal = document.querySelector('.bi-bi-list');
```

Comment:
Acho que faz mais sentido mudar o nome dessa classe pra `open-icon`
```suggestion
const open_modal = document.querySelector('.open-icon');
```

Reply:
Não é uma prática comum usar `snake case` (usa underline) em Javascript, normalmente a gente usa `camel case`

Reply:
Bem melhor, eu vacieli no camel case.

Reply:
Tá feito.

Reply:
Acho que vc deve ter feito em outro branch pq as mudanças não estão aparecendo aqui.